### PR TITLE
Upgrade FontAwesome from 4.7.0 to 5.0.13

### PR DESCRIPTION
**Pull request by lightnet328**
# About

Upgrade FontAwesome from 4.7.0 to 5.0.13 for using new icons.
The appearance of icon was slightly changed because FontAwesome version 5 has been re-written and re-designed completely from scratch.

# How to upgrade

Changed icon names list was provided (https://fontawesome.com/how-to-use/upgrading-from-4#icon-name-changes).
However We can't know whether used icon names has changed.
Therefore I checked out by following ways.

1. Open https://fontawesome.com/icons?d=gallery&m=free
2. Load all icons
3. Open DeveloperTools
4. Run following a script and copy output
```
console.log(Array.from(document.querySelectorAll("#results-icons ul li div a i")).map(e => e.className.split(" ")[1]).join("\n"));
```
5. Run following commands
```
$ cat > free-icons (Command + V, Enter, Ctrl + C)
$ comm <(grep -hrwo ./lib/views -e "fa-[a-z-]*" | sort | uniq) <(cat ./free-icons | sort | uniq) -2 -3
fa-envelope-o
fa-file-text
fa-fw
fa-gears
fa-meh-o
fa-pencil
fa-pencil-square-o
fa-pulse
fa-sign-out
fa-smile-o
fa-star-o
fa-thumbs-o-up
fa-trash-o
fa-warning
```
6. Look for the corresponding new icons...

I searched manually changed icon, but some icons may not be found.

### CODE REVIEW

**Comment by sotarok** [comment]
I changed the base branch to `master`.

File: client/components/Common/Icon.js [hunk 1]
```diff
@@ -5,25 +5,33 @@ export default class Icon extends React.Component {
 
   render() {
     const name = this.props.name || null;
-    const isSpin = this.props.spin ? ' fa-spinner' : '';
+    const isSpin = this.props.spin ? ' fa-spinner fa-pulse' : '';
```
Comment (hiroppy) [inline client/components/Common/Icon.js]: Do we need the space?

```diff
- const isSpin = this.props.spin ? ' fa-spinner fa-pulse' : '';
+ const isSpin = this.props.spin ? 'fa-spinner fa-pulse' : '';
```

L17: 
```js
<i className={`fa${type} fa-${name} ${isSpin}`} />
```

Comment (lightnet328) [reply client/components/Common/Icon.js]: It's unnecessary. I removed it.

**Comment by sotarok** [comment]
Found missed some icons:

Google Icon
image

Also
image

**Comment by lightnet328** [comment]
I have completely overlooked it.
I fixed missing icons now.

**Pull request merged**